docs/src/test/java/docs/http/javadsl/server/HttpServerExampleDocTest.java

```diff
@@ -321,6 +322,35 @@ void discardEntityManuallyCloseConnections() {
       );
         //#discard-close-connections
       }
-    
-  
+
+  public static void gracefulTerminationExample() throws Exception {
+    //#graceful-termination
+    ActorSystem system = ActorSystem.create();
+    Materializer materializer = ActorMaterializer.create(system);
+
+
+    CompletionStage<ServerBinding> binding = Http.get(system).bindAndHandle(
+        Directives.complete("Hello world!").flow(system, materializer),
+        ConnectHttp.toHost("localhost", 8080), materializer);
+
+    ServerBinding serverBinding = binding.toCompletableFuture().get(3, TimeUnit.SECONDS);
```

Comment:
Don't do this at home. We should not show such code in documentation, it will be copied.

Reply:
We do show this in many places in the "start a server and wait for binding" so I did so too.
Can go with nesting but it makes it rather weird tbh

Reply:
ok